execution/src/windows/execution.cc

```diff
@@ -117,11 +118,11 @@ namespace leatherman { namespace execution {
 
         // Format a name for the pipe based on the process and counter
         wstring name = boost::nowide::widen((boost::format("\\\\.\\Pipe\\leatherman.%1%.%2%") %
-            GetCurrentProcessId() %
+            GetCurrentThreadId() %
```

Comment:
I suppose the rand name introduces an (almost negligible) possibility for name collisions (depending on how random it is), where the sequential counter will avoid it.

Process + Thread id makes sense though.  Improvement for the future?